src/application/machines.ts

```diff
@@ -36,22 +34,14 @@ type State = {
 
 type Actions = {
   connectToClient: StateMachine.ActionFunction<Context, Events>;
+  cancelRequestInterval: StateMachine.ActionFunction<Context, Events>;
   startRequestInterval: StateMachine.ActionFunction<Context, Events>;
-  unsubscribeFromAll: StateMachine.ActionFunction<Context, Events>;
 };
 
 export function createDevtoolsMachine({ actions }: { actions: Actions }) {
   return createMachine<Context, Events, State>(
     {
       initial: "initialized",
-      context: {
```

Comment:
I'm removing `context` on the machine because we never really kept this up-to-date with the latest values from the client, which means we weren't really using this as a source-of-truth. The only time we read this context was when we initialized the panel so that we could show data quickly. That has now been updated to use the rpc client to pull this data instead.